src/main/java/dan200/computercraft/ComputerCommand.java

```diff
@@ -0,0 +1,44 @@
+package dan200.computercraft;
+
+import dan200.computercraft.shared.computer.core.ComputerFamily;
+import dan200.computercraft.shared.computer.core.ServerComputer;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import org.apache.commons.lang3.ArrayUtils;
+
+public class ComputerCommand extends CommandBase {
+
+    @Override
+    public String getName() {
+        return "computer";
+    }
+
+    @Override
+    public String getUsage(ICommandSender iCommandSender) {
+        return "computer <id> <value1> [value2]...";
+    }
+
+    @Override
+    public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
+        if( args.length < 2 ){
+            throw new CommandException( "Usage: /computer <id> <value1> [value2]..." );
+        }
+        try {
+            ServerComputer computer = ComputerCraft.serverComputerRegistry.lookup(Integer.valueOf(args[0]));
```

Comment:
Having another think about this, I think it would be better to do find *all* computers with the given ID, queuing for those which are commands. If no such computers are found then it'd error. Most of the time they'll only be one computer with the given ID, but some programs (*cough* Taken *cough*) spawn multiple computers with the same ID which makes this unusable.